requirements/requirements.txt

```diff
@@ -57,3 +57,4 @@ opentelemetry-sdk==1.27.0
 opentelemetry-exporter-otlp==1.27.0
 opentelemetry-instrumentation-requests==0.48b0
 opentelemetry-instrumentation-flask==0.48b0
+opentelemetry-distro==0.48b0
```

Comment:
Should we make the open telemetry optional?

Reply:
It is optional but we need to install this library still.